README.md

```diff
@@ -65,9 +68,11 @@ Through this repository, we want to put out a list of curated resources which he
 - [Skrapp](https://www.skrapp.io/): Browser addon to find emails on Linkedin  
 - [Email Extractor](https://chrome.google.com/webstore/detail/email-extractor/jdianbbpnakhcmfkcckaboohfgnngfcc?hl=en): Chrome extension to extract emails from web pages  
 - [Convertcsv](http://convertcsv.com/email-extractor.htm): Online tool to extract email addresses in text, web pages, data files etc. 
+- [Voilanorbert](https://www.voilanorbert.com) - extensive email finder service
```

Comment:
This asks for Login before giving out free info. We will be removing other such services too, shortly.

Reply:
Oh, that's true. fucked up, you can replace it with hunter.io. It gives you around 50 free searches